tfe/resource_tfe_terraform_version.go

```diff
@@ -134,3 +135,21 @@ func resourceTFETerraformVersionDelete(d *schema.ResourceData, meta interface{})
 
 	return nil
 }
+
+func resourceTFETerraformVersionImporter(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	tfeClient := meta.(*tfe.Client)
+
+	// Splitting by '-' and checking if the first elem is equal to tool
+	// determines if the string is a tool version ID
+	s := strings.Split(d.Id(), "-")
+	if s[0] != "tool" {
+		versionID, err := fetchTerraformVersionID(d.Id(), tfeClient)
+		if err != nil {
+			return nil, fmt.Errorf("error retrieving terraform version %s: %v", d.Id(), err)
```

Comment:
💅 use %w when wrapping other errors with fmt.Errorf

Reply:
Huh was not aware of this, always defaulted to %v! Will add thanks!